uni/lib/view/library/widgets/library_reservations_card.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:uni/model/entities/library_reservation.dart';
+import 'package:uni/model/providers/library_reservations_provider.dart';
+import 'package:uni/model/request_status.dart';
+import 'package:uni/utils/drawer_items.dart';
+import 'package:uni/view/common_widgets/generic_card.dart';
+import 'package:uni/view/library/widgets/reservation_row.dart';
+
+class LibraryReservationsCard extends GenericCard {
+  LibraryReservationsCard({Key? key}) : super(key: key);
+
+  const LibraryReservationsCard.fromEditingInformation(
+      Key key, bool editingMode, Function()? onDelete)
+      : super.fromEditingInformation(key, editingMode, onDelete);
+
+  @override
+  onClick(BuildContext context) => Navigator.pushNamed(
+      context, '/${DrawerItem.navLibraryReservations.title}');
+
+  @override
+  String getTitle() => 'Gabinetes Reservados';
+
+  @override
+  Widget buildCardContent(BuildContext context) {
+    return Consumer<LibraryReservationsProvider> (
```

Comment:
This logic should be done using a RequestDependentWidget

Reply:
This is more of a suggestion and not required, but maybe on the cardWidget do a shimmer effect while loading, to make it more consistent between cards on the Main Page, because schedule and exams already have it.